src/test/scenarios/attestation/output/src/attestation/README.md

```diff
@@ -1,5 +1,32 @@
-Microsoft Azure CLI 'attestation' Extension
-==========================================
+# Azure CLI attestation Extension #
+This is a extension for attestation
 
-This package is for the 'attestation' extension.
-i.e. 'az attestation'
+### How to use ###
+Install this extension using the below CLI command
+```
+az extension add --name attestation
+```
+
+### Included Features ###
+#### attestation: ####
+attestation
+* Examples:*
```

Comment:
could we make resource names consistent through the whole scenarios?
e.g. use `MyResourceGroup` for all --resource-group

Reply:
The name is selected from example. "myattestationprovider" is used in example

Reply:
Talked offline, It depends on quality of example